calico/operations/troubleshoot/vpp.mdx

```diff
@@ -113,23 +112,23 @@ calivppctl vppctl myk8node1
 To list existing interfaces and basic counters use
 
 ```
```

Comment:
```suggestion
```bash
```

Reply:
This should be done throughout.

Reply:
Doing so would **italicize** the text after the "vpp#" and thats why I have used "bash" in some places and not used in some other places.